packages/v4/src/content/design-guidelines/components/select/select.md

```diff
@@ -9,7 +9,29 @@ related: [
 ]
 ---
 
-**Select lists** enable users to select one or more items from a list. Select lists should also be used when options are dynamic or variable. 
+**Select lists** enable users to select one or more items from a list. Select lists should also be used when options are dynamic or variable.
+
+## Elements and variations
+
+### Basic select
+A select component is a type of dropdown menu that consists of a toggle control to open and close a menu of actions or links. Selects differ from dropdowns in that they persist selection where dropdowns are typically used to present a list of actions or links.
+
+<img src="./img/basic-select.png" alt="basic select menu" width="303"/>
+
+1. **Toggle:** The toggle is used to open and close the menu. It takes on all of the behaviors of a button. The toggle label always reflects the currently selected item or may hold a placeholder if nothing is selected.
+
+2. **Menu:** The menu contains a list of options or values. Menus should optimally contain seven (7) or less items for easy scanning by the user. For longer menus, items may be grouped. See the [Menu component guidelines](/components/menu/design-guidelines) for more information about supported options, including use of icons, grouping, and descriptions. The Dropdown component includes many common built-in menu variants. It’s also possible to attach a custom menu by using the [Basic panel variant](/components/select/react#panel).
+
+### Select variations
```

Comment:
From here to the usage header repeats a lot of what's already been added to the guidelines. I would definitely just merge them somehow

Reply:
Or maybe delete this who select variations - it's a little confusing too because typeahead and checkbox do very different things ! If anything i'd expected a checkbox vs multiple select comparison?

Reply:
I'm going to just delete this section. I added it at the last minute, but I see where it is redundant. I mainly wanted to just point out the basic elements and reference the menu guidelines as I do in the first section.